src/Microsoft.AspNetCore.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -134,6 +134,10 @@ public static IWebHostBuilder ConfigureLogging(this IWebHostBuilder hostBuilder,
         /// </summary>
         /// <param name="hostBuilder">The <see cref="IWebHostBuilder" /> to configure.</param>
         /// <param name="configureLogging">The delegate that configures the <see cref="ILoggerFactory"/>.</param>
+        /// <typeparam name="T">
+        /// The type of <see cref="ILoggerFactory"/> to configure.
+        /// The delegate will not execute if the type provided does not match the <see cref="ILoggerFactory"/> used by the <see cref="IWebHostBuilder"/>
```

Comment:
Fixed in: https://github.com/aspnet/Hosting/commit/c812e9ae6fef3a533e99c2bbf569d42134d3e1c1